Makefile

```diff
@@ -36,6 +37,7 @@ EXECUTABLE8=$(TARGET8:%.cpp=$(BINDIR)/%.app)
 EXECUTABLE9=$(TARGET9:%.cpp=$(BINDIR)/%.app)
 EXECUTABLE10=$(TARGET10:%.cpp=$(BINDIR)/%.app)
 EXECUTABLE11=$(TARGET11:%.cpp=$(BINDIR)/%.app)
+EXECUTABLE12=$(TARGET12:%.cpp=$(BINDIR)/%.app)
```

Comment:
Yeah, I find this Makefile has a lot of repetition.  I think it could be cleaned up.

Alternately I could try to make a cmake build file instead, if we want to try something more modern.